detox/src/devices/drivers/android/genycloud/helpers/GenyCloudInstanceLauncher.test.js

```diff
@@ -54,8 +54,10 @@ describe('Genymotion-Cloud instance launcher', () => {
       try {
         const instance = anInstance();
         await uut.shutdown(instance);
-        fail('Expected an error');
-      } catch (e) {}
+      } catch (e) {
+        return;
+      }
+      throw new Error('Expected an error!');
```

Comment:
why not `await expect(uut.shutdown(instance)).rejects.toThrow(/my error desc/)` ?